selfdrive/car/tests/test_lateral_limits.py

```diff
@@ -66,26 +65,8 @@ def calculate_0_5s_jerk(control_params, torque_params):
   def test_jerk_limits(self):
     up_jerk, down_jerk = self.calculate_0_5s_jerk(self.control_params, self.torque_params)
     car_model_jerks[self.car_model] = {"up_jerk": up_jerk, "down_jerk": down_jerk}
-    self.assertLessEqual(up_jerk, MAX_LAT_JERK_UP + MAX_LAT_JERK_UP_TOLERANCE)
-    self.assertLessEqual(down_jerk, MAX_LAT_JERK_DOWN)
+    assert up_jerk <= MAX_LAT_JERK_UP + MAX_LAT_JERK_UP_TOLERANCE
+    assert down_jerk <= MAX_LAT_JERK_DOWN
 
   def test_max_lateral_accel(self):
```

Comment:
@adeebshihadeh this report was useful

Reply:
my bad, didn't notice it got removed

Reply:
https://github.com/commaai/openpilot/pull/32614